src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -525,9 +525,15 @@ internal void ExecutePackageDownload(string name, PackageVersion version, string
 
                 // determine if any of the packages contain binaries or python scripts.  
                 var containsBinariesOrPythonScripts = dependencyVersionHeaders.Any(x =>
-                    x.contains_binaries ||
-                    x.contents.Contains(PackageManagerClient.PackageContainsBinariesConstant) ||
-                    x.contents.Contains(PackageManagerClient.PackageContainsPythonScriptsConstant));
+                {
+                    var are_contents_empty = string.IsNullOrEmpty(x.contents);
+                    var contains_binaries = x.contains_binaries ||
+                                            !are_contents_empty && x.contents.Contains(PackageManagerClient.PackageContainsBinariesConstant);
+                    var contains_python =
+                        !are_contents_empty && x.contents.Contains(PackageManagerClient.PackageContainsPythonScriptsConstant);
+                    return contains_binaries || contains_python;
```

Comment:
`PackageManagerClient.PackageContainsPythonScriptsConstant` and `PackageManagerClient.PackageContainsBinariesConstant` are deprecated so I'm not sure if we should use these or something else.

Reply:
I'm not sure either, but the package header only contains one boolean field - contains binaries which is true if the package contains binaries or python...

Reply:
There are 41 versions across 22 packages that are still using this "obsolete" method of marking packages.  We could theoretically update those version records in the database to have the `contains_binaries` field set to `true` and then we could remove these constants.

Reply:
Is there a `contains_python` field as well? If so, then we need to add it to the check here.

Reply:
No, there is only a single field for both.